flattentool/__init__.py

```diff
@@ -208,14 +208,15 @@ def unflatten(input_name, base_json=None, input_format=None, output_name=None,
         base[root_list_path] = list(result)
 
     if xml:
+        xml_root_path = base_configuration.get('XMLRootPath', 'iati-activities')
```

Comment:
I'm wondering whether we should remove this default entirely.

Reply:
Ohhhhh… In fact, it’s actually a bit tricky and relies on #176, I think. (That’s possibly why I reffed #176 in the PR, I don’t remember.)

Can we revisit after #176 is sorted?

Reply:
> Can we revisit after #176 is sorted?

Yes, that makes sense.